### DISCUSSION TITLE
"Shortcuts" in Prompt choices

### DISCUSSION THREAD

**Discussion by mickvikt** [spectreconsole/spectre.console#172] [Ideas]
Hello,
I wanted to ask if there is any way to specify a one-letter shortcut that can be entered instead of the complete choice name, akin to some Linux console interfaces, e.g.

Choose your action: [P]roceed/[C]ancel,

so that P/p/Proceed/proceed would be accepted for the choice "Proceed" and C/c/Cancel/cancel for the choice of "Cancel"?

Thank you.

**Comment by patriksvensson** [comment]
@mickvikt There's currently no support for this but could be added.
